### PR TITLE
feature: Add a way to turn on debug logging and fix scalafix warmup

### PR DESCRIPTION
**Pull request by tgodzik**
Seems the main idea behind scribe is to make it easy to define in the code, so I decided to add an new metals property.

One issue is that Flyway debug logs still seem to end up in whichever handler is defined first, but this should not be an issue in most cases, since those logs are not super interesting.

### CODE REVIEW

File: metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala [hunk 1]
```diff
@@ -48,14 +48,14 @@ case class ScalafixProvider(
   // Warms up the Scalafix instance so that the first organize imports request responds faster.
   def load(): Unit = {
     if (!Testing.isEnabled) {
+      val tmp = workspace
```
Comment (ckipp01) [inline metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala:51]: What was actually happening here? How come moving it out of the `try` fixes it?

**Review by ckipp01 — APPROVED** [review]
Ah nice, I always wondered about debug for scribe.

File: metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala (hunk 1 above)
Comment (tgodzik) [reply metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala:51]: Not 100 percent sure 😅 it seems there was a race, but how I have no idea 😑

**Comment by darkfrog26** [comment]
Let me know if there's something I could add to Scribe that would assist in making this easier.

File: metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala (hunk 1 above)
Comment (tgodzik) [reply metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala:51]: I just added additional checks to see if evaluation indeed finishes successfully and it seems it does. There might be some magic behind the finally block.

**Pull request merged**